### PR TITLE
Improve get_ufunc_kernel

### PR DESCRIPTION
**Pull request by okuta**
Remove `util.memoize`.

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

File: cupy/core/_kernel.pyx [hunk 1]
```diff
@@ -596,10 +596,9 @@ cdef class ElementwiseKernel:
         return kern
 
 
-@util.memoize(for_each_device=True)
-def _get_ufunc_kernel(
-        in_types, out_types, routine, args_info, params, name, preamble,
-        loop_prep):
+cdef function.Function _get_ufunc_kernel(
```
Comment (asi1024) [inline cupy/core/_kernel.pyx]: Is it OK to remove memoization?

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by asi1024** [comment]
Jenkins failure is irrelevant to this PR. LGTM.

**Pull request merged**